external/jenkins/README.md

```diff
@@ -0,0 +1,21 @@
+## External Jenkins Tests
+
+Jenkins tests are part of the external third-party application tests that help verify that the AdoptOpenJDK binaries are good by running a variety of Java applications inside of Docker containers. AdoptOpenJDK/openjdk-tests/Issue #172 lists the applications that we have initially targeted to best exercise the AdoptOpenJDK binaries. For each application, we choose to run a selection of their functional tests.
+
+## Running External Jenkins tests locally
+
+To run any AQA tests locally, you follow the same pattern:
+
+0. Ensure your test machine is set up with [test prereqs](https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/doc/Prerequisites.md). For external tests, you do need Docker installed.
+1. Download/unpack the SDK that you want to test to your test machine
+1. `export TEST_JDK_HOME=</pathToWhereYouInstalledSDK>`
+1. `git clone https://github.com/AdoptOpenJDK/openjdk-tests.git`
+1. `cd openjdk-tests`
+1. `./get.sh`
+1. `cd TKG`
+1. export required environment variables, BUILD_LIST and EXTRA_DOCKER_ARGS (`export BUILD_LIST=external/jenkins> and <export EXTRA_DOCKER_ARGS="-v $TEST_JDK_HOME:/opt/java/openjdk"`)
+1. `make compile` (This fetches test material and compiles it, based on build.xml files in the test directories) 1.`make _jenkins_test` (When you defined BUILD_LIST to point to a directory in aqa-tests/external, then this is a testCaseName from the playlist.xml file within the directory you chose).
```

Comment:
Please put 1. `make _jenkins_test` on its own line so it shows up as a numbered bullet point in the list.